SearchInSorterArrUnknownSize.java

```diff
@@ -0,0 +1,32 @@
+// Time Compexity : O(log n)
```

Comment:
What is n?

Reply:
it should actually be O (Log mn)

Reply:
O(log n) in range high- low

Reply:
every time you have to mention the variables you are using for your complexities, here n is not present in code and you have mentioned it in TC so you have to explicitly tell what n is. 
For example here 
TC is O(log n) where n is value of the high variable